dev.tfkhdyt.pomodoro.yml

```diff
@@ -0,0 +1,42 @@
+id: dev.tfkhdyt.pomodoro
+
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+
+command: pomodoro
+finish-args:
+  - --socket=wayland # Permission needed to show the window
+  - --socket=fallback-x11 # Permission needed to show the window
+  - --device=dri # OpenGL, not necessary for all projects
+  - --share=ipc
+  - --socket=pulseaudio
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.portal.Desktop
+
+modules:
+  - name: binary
+    buildsystem: simple
+    sources:
+      - type: file
+        # path: src-tauri/target/release/bundle/deb/Pomodoro_0.9.2_amd64.deb
+        url: https://github.com/tfkhdyt/pomodoro/releases/download/v0.9.2/Pomodoro_0.9.2_amd64.deb
```

Comment:
why not build from source?

Reply:
so what happen when building the aarch64 package?